### PR TITLE
Store stdout otuput into a CSV file.

### PR DESCRIPTION
**Pull request by protobird-git**
1) Output delimiters separately for easier csv processing.
2) In csv output, split duration stats into 2, one in units, one in percentage.

### CODE REVIEW

**Review by okkwon — APPROVED** [review]
Looks great. Let's change `output` by `os` to follow the LLVM naming convention.

File: iree-prof-tools/iree-prof-output-stdout.h [hunk 1]
```diff
@@ -39,12 +44,16 @@ class IreeProfOutputStdout : public IreeProfOutput {
...
+  // Interface of output stream to replace stdout. Used internally.
+  class OutputStream;
+
  private:
   const bool output_zone_stats_;
   const bool output_per_op_stats_;
   const std::regex zone_regex_;
   const std::regex thread_regex_;
   const DurationUnit unit_;
+  const std::unique_ptr<OutputStream> output_;
```
[the remark this answers is not in the document]
Comment (protobird-git) [reply iree-prof-tools/iree-prof-output-stdout.h]: Done

File: iree-prof-tools/iree-prof-output-stdout.cc [hunk 2]
```diff
@@ -358,52 +546,59 @@ IreeProfOutputStdout::IreeProfOutputStdout(bool output_zone_stats,
...
+                 << "not empty.";
+}
 
 IreeProfOutputStdout::~IreeProfOutputStdout() = default;
 
 absl::Status IreeProfOutputStdout::Output(tracy::Worker& worker) {
-  std::cout << "[TRACY    ] Capture Name: " << worker.GetCaptureName() << "\n";
-  std::cout << "[TRACY    ]     Cpu Arch: " << ArchToString(worker.GetCpuArch())
-            << "\n";
+  *output_ << "[TRACY    ]" << " Capture Name" << ": "
```
[the remark this answers is not in the document]
Comment (protobird-git) [reply iree-prof-tools/iree-prof-output-stdout.cc]: Done

**Review by okkwon — APPROVED** [review]
Looks great. Thanks

**Pull request merged**